website/content/docs/nia/terraform-modules.mdx

```diff
@@ -31,21 +31,23 @@ See below sections for more information on [defining source input](#source-input
 
 ### Source Input
 
-A source input allows for a task to satisfy the input requirements defined by the Terraform Module’s [input variables](https://www.terraform.io/docs/language/values/variables.html), and is configured alongside a task’s condition. Both the source input and condition define objects to be monitored, but for differing reasons.
+CTS monitors defined objects to provide source input to a task. This source input satisfies the requirements defined by the task's Terraform Module's [input variables](https://www.terraform.io/docs/language/values/variables.html).
 
-The condition defines monitored objects with criteria. When this criteria is satisfied, CTS will then trigger a task.
+A task's source input slightly defers the task's condition even though both monitor defined objects. The task's condition monitors defined objects with a configured criteria. When this criteria is satisfied, the task will trigger.
 
-The source input however, defines monitored objects with the intent of providing values or metadata about these objects to the Terraform Module. The source input and condition objects can be the same, such as when `task.services` is provided without a source input block and condition block, but they can also be defined using separate blocks. In this way the source input does not need to be tied to the provided condition in order to satisfy the Terraform Module.
+The source input however, monitors defined objects with the intent of providing values or metadata about these objects to the Terraform Module. The monitored source input and condition objects can be the same object, such as a task is configured with a `condition "services"` block and `source_includes_var` is set to `true`. The source input and condition can be different objects and configured separately, such as a task configured with a `condition "catalog-services` and `source_input "consul-kv"` block. In this way the source input does not need to be tied to the provided condition in order to satisfy the Terraform Module.
```

Comment:
```suggestion
The source input however, monitors defined objects with the intent of providing values or metadata about these objects to the Terraform Module. The monitored source input and condition objects can be the same object, such as a task is configured with a `condition "services"` block and `source_includes_var` is set to `true`. The source input and condition can be different objects and configured separately, such as a task configured with a `condition "catalog-services` and `source_input "consul-kv"` block. This results in the monitored source input not needing to be tied to the provided condition in order to satisfy the Terraform Module.
```
Does the change still make sense? I tried to clean up the language.

Reply:
Yes! Makes sense, thank you